src/vivarium_public_health/disease/state.py

```diff
@@ -302,81 +346,56 @@ def _cleanup_effect(self, index, event_time):
         if self.cleanup_function is not None:
             self.cleanup_function(index, event_time)
 
-    def compute_disability_weight(self, index):
-        """Gets the disability weight associated with this state.
-
-        Parameters
-        ----------
-        index : iterable of ints
-            An iterable of integer labels for the simulants.
-
-        Returns
-        -------
-        `pandas.Series`
-            An iterable of disability weights indexed by the provided `index`."""
-        population = self.population_view.get(index)
-
-        return self._disability_weight(population.index) * ((population[self._model] == self.state_id)
-                                                            & (population.alive == 'alive'))
-
-    def __repr__(self):
-        return 'DiseaseState({})'.format(self.state_id)
+    def load_prevalence_data(self, builder):
+        if 'prevalence' in self._get_data_functions:
+            return self._get_data_functions['prevalence'](self.cause, builder)
+        else:
+            return builder.data.load(f"{self.cause_type}.{self.cause}.prevalence")
 
+    def load_birth_prevalence_data(self, builder):
+        if 'birth_prevalence' in self._get_data_functions:
+            return self._get_data_functions['birth_prevalence'](self.cause, builder)
+        else:
+            return 0
 
-class TransientDiseaseState(BaseDiseaseState, Transient):
+    def load_dwell_time_data(self, builder):
+        if 'dwell_time' in self._get_data_functions:
+            dwell_time = self._get_data_functions['dwell_time'](self.cause, builder)
+        else:
+            dwell_time = 0
 
-    def __repr__(self):
-        return 'TransientDiseaseState(name={})'.format(self.state_id)
+        if isinstance(dwell_time, pd.Timedelta):
+            dwell_time = dwell_time.total_seconds() / (60 * 60 * 24)
+        if (isinstance(dwell_time, pd.DataFrame) and np.any(dwell_time.value != 0)) or dwell_time > 0:
+            self.transition_set.allow_null_transition = True
 
+        return dwell_time
 
-class MortalityEffect:
```

Comment:
Like the special disease, all disease states now have csmr/excess mortality.  It is appropriately set to zero when necessary.